### DISCUSSION TITLE
Problem starting LLM

### DISCUSSION THREAD

**Discussion by WillMiles** [undreamai/LLMUnity#248] [Q&A]
Hey Antonis, got another problem after updating to the new version. It seems similar to the previous Windows-specific issues previously discussed but the changes don't seem to be fixing it. I've also tried reverting to undreamai-v1.1.10 just in case which didn't seem to help.

What's happening is that the LLM service never successfully starts. It sends the command to start it and then...nothing. If I try to reference the LLM it either throws a million errors (in the editor) or simply crashes (in the build), even while checking `LLM.modelSetupComplete` before doing anything with the LLM. Additionally, if I run it in the editor and then stop it never fully stops, I have to manually close and reopen the editor, which as you can guess is extremely frustrating. 

I've tried restarting/reinstalling several times to no avail.

Any advice would be much appreciated, here is the Editor.Log when trying to run the game:

https://pastebin.com/mKyKdwU5

Thanks again for all the work you've done with this package.

**Comment by WillMiles** [comment]
Seems to inconsistently also occur in builds on other computers--even if the AI works in the build (and the editor) for some computers, the same exact build on different computers the LLM service won't even start.

**Comment by amakropoulos** [comment]
hi! Could you send me the specs (CPU/GPU/RAM) and the player logs from these computers?

**Comment by amakropoulos** [comment]
it would be very helpful if we could do some live troubleshooting over Discord

**Reply by Eelguezabal** [reply]
Hi, Will and I are coworkers, he is on vacation currently, the issues he posted here are essentially the same ones we are discussing here: https://github.com/undreamai/LLMUnity/issues/249
